benchmark/syclblas/blas3/trsm.cpp

```diff
@@ -108,9 +111,35 @@ void run(benchmark::State& state, ExecutorType* executorPtr, char side,
 
   // Measure
   for (auto _ : state) {
-    // Run
+    auto start = std::chrono::system_clock::now();
+    auto events = blas_method_def();
+    auto end = std::chrono::system_clock::now();
+    double overall_time = (end - start).count();
```

Comment:
Would ```std::chrono::high_resolution_clock``` not be preferable for this?

Reply:
Yes, however, the `system_clock` in https://github.com/codeplaysoftware/sycl-blas/blob/2e0e5e90e27b393c9adf47c5ec26a8376158e4fc/benchmark/common_utils.hpp#L625) so I kept the same here.

We could add this to the internal discussion on sycl-blas

Reply:
That's fair, I think it's fine to leave it as is then.